package.json

```diff
@@ -268,41 +270,40 @@
     "swagger-ui-express": "4.3.0",
     "swr": "1.2.0",
     "testcontainers": "7.23.2",
-    "timeout-signal": "1.1.0",
     "to-querystring": "1.1.1",
     "triple-beam": "1.3.0",
     "tslib": "1.14.1",
     "unified": "9.2.0",
     "uuidv4": "6.0.8",
+    "whatwg-fetch": "3.6.2",
     "winston": "3.7.2",
     "winston-cloudwatch": "3.1.1",
     "winston-transport": "4.5.0",
     "xlsx": "0.17.0",
     "xml2js": "0.4.23",
     "xstate": "4.20.0",
     "yargs": "15.4.1",
-    "yup": "0.29.0",
     "zod": "1.11.13"
   },
   "devDependencies": {
-    "@anatine/esbuild-decorators": "^0.2.4",
+    "@anatine/esbuild-decorators": "0.2.4",
     "@anatine/esbuildnx": "0.2.0",
     "@aws-sdk/client-ses": "3.145.0",
     "@aws-sdk/client-ssm": "3.49.0",
     "@babel/core": "7.18.2",
     "@babel/plugin-proposal-decorators": "7.18.2",
     "@babel/plugin-transform-runtime": "7.18.2",
     "@babel/preset-env": "7.18.2",
-    "@babel/preset-react": "7.17.12",
-    "@babel/preset-typescript": "7.17.12",
+    "@babel/preset-react": "7.18.6",
+    "@babel/preset-typescript": "7.18.6",
     "@gperdomor/nx-docker": "0.2.3",
-    "@graphql-codegen/cli": "1.17.8",
-    "@graphql-codegen/fragment-matcher": "1.17.8",
-    "@graphql-codegen/schema-ast": "1.17.8",
-    "@graphql-codegen/typescript": "1.17.8",
-    "@graphql-codegen/typescript-operations": "1.17.15",
-    "@graphql-codegen/typescript-react-apollo": "1.17.7",
-    "@graphql-codegen/typescript-resolvers": "1.17.9",
+    "@graphql-codegen/cli": "2.12.0",
+    "@graphql-codegen/fragment-matcher": "3.3.1",
+    "@graphql-codegen/schema-ast": "2.5.1",
+    "@graphql-codegen/typescript": "2.7.3",
+    "@graphql-codegen/typescript-operations": "2.5.3",
+    "@graphql-codegen/typescript-react-apollo": "3.3.3",
+    "@graphql-codegen/typescript-resolvers": "2.7.3",
```

Comment:
These are used by our schema generation script. No regressions? 🙈

Thankfully TypeScript should catch most of it.

Reply:
We are generating schemas successfully but I'm not sure how to test them specifically beyond that, would love your input in that area.